Changelogs for Digital Twin Manager UI 1021.0.0 (#2833)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1021-0-0-update-subassets-contextual-help-to-navigate-to-dtm-documentation.md

---
date: ""

> title: "Update Subassets contextual help to navigate to DTM documentation"

Reviewer: ```suggestion
title: "Updated link to the documentation in contextual help of Subassets page"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-1592"
version: "1021.0.0"
---

> Previously the contextual help URL in subassets screen is pointing to the cockpit documentation. Now subassets is added as part of dtm-plugins and the URL will be navigated to DTM documentation

Reviewer: ```suggestion
Previously, the link in the contextual help on the **Subassets** page pointed to the Cockpit documentation. This URL has been changed to point to the Digital Twin Manager application where subassets are described as part of the dtm-plugin.
```
